React/Views/RefreshControl/RCTRefreshControl.m

```diff
@@ -41,6 +41,12 @@ - (void)layoutSubviews
 {
   [super layoutSubviews];
 
+  // Fix for bug #7976
+  // TODO: Remove when updating to use iOS 10 refreshControl UIScrollView prop.
```

Comment:
We should update this comment right? This class uses the scrollview prop.

Reply:
Can't say I understand the root cause ... probably worth pinging @GantMan about that comment as he wrote the original fix, https://github.com/facebook/react-native/pull/9249

Reply:
@GantMan just another ping here as I know we're barreling towards a 0.64 final release and don't want to allow it out without this regression fixed.

Reply:
Also pinging @janicduplessis as it seems they were involved in that discussion: https://github.com/facebook/react-native/pull/9249#discussion_r76986319